flink-core/src/main/java/org/apache/flink/api/common/eventtime/WatermarkStrategy.java

```diff
@@ -223,8 +223,8 @@ static <T> WatermarkStrategy<T> forMonotonousTimestamps() {
     /**
      * Creates a watermark strategy for situations where records are out of order, but you can place
      * an upper bound on how far the events are out of order. An out-of-order bound B means that
-     * once the an event with timestamp T was encountered, no events older than {@code T - B} will
-     * follow any more.
+     * once an event with timestamp T was encountered, no events older than {@code T - B} will
+     * follow anymore.
```

Comment:
This text seems to be almost an incorrect copy of the [class javadoc ](https://nightlies.apache.org/flink/flink-docs-release-1.20/api/java/org/apache/flink/api/common/eventtime/BoundedOutOfOrdernessWatermarks.html).

I think a link to the class javadoc, with the existing `@see` or similar and then javadoc including the `@param` would seem more appropriate. Maybe something like

`Creates a watermark strategy for @see BoundedOutOfOrdernessWatermarks using the supplied out-of-order bound`
@param maxOutOfOrderness out-of-order bound duration.